### PR TITLE
Improve README file

### PR DESCRIPTION
**Pull request by anavarre**

### CODE REVIEW

**Comment by anavarre** [comment]
Disclaimer: this is primarily a lame excuse to test this project (it works!)

We could improve the README file by helping 2FA users and also recommending to encrypt the $HOME directory for security.

File: README.rst [hunk 1]
```diff
@@ -55,4 +60,5 @@ calling `hub pull-request` would open a new pull-request.
 
 git-pull-request wraps all those operation in a single hand convenient tool.
 
+.. _token: https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/
```
Comment (jd) [inline README.rst]: You sure about the syntax here? seems to me the full string should be repeated.

Comment (anavarre) [inline README.rst]: First time I have to interact with the reStructuredText format. Still trying to understand how to create a properly formatted URL here.

**Comment by anavarre** [comment]
Okay, forced push a different way of formatting the URL based on https://raw.githubusercontent.com/jakubroztocil/httpie/master/README.rst - Let me know if that works for you.

**Comment by jd** [comment]
You don't have to `git push -f` by yourself, `git pull-request` can do it for you ;)

**Comment by anavarre** [comment]
> You don't have to git push -f by yourself, git pull-request can do it for you ;)

Still learning it! :-)

**Pull request merged**